include/syscall_handler.h

```diff
@@ -539,7 +539,12 @@ static inline int z_obj_validation_check(struct z_object *ko,
 #define Z_SYSCALL_OBJ_NEVER_INIT(ptr, type) \
 	Z_SYSCALL_IS_OBJ(ptr, type, _OBJ_INIT_FALSE)
 
+__syscall void sys_cache_flush(void *addr, size_t size);
+
+__syscall void sys_cache_invd(void *addr, size_t size);
+
```

Comment:
aren't these functions declared in `include/cache.h` ? It seems that your re-declaring them.

Reply:
Those are the API names and are declared in include/cache.h. These are the system calls. Will these not be considered different? Although the names are same.

Reply:
Make something a syscall needs to be done where the API is declared, in this case in include/cache.h AFAIK.

Reply:
Addressed this.